configparser_test.go

```diff
@@ -114,12 +114,16 @@ func (s *ConfigParserSuite) TestSaveWithDelimiterAndDefaults(c *C) {
 
 // ParseFromReader() parses the Config data from an io.Reader.
 func (s *ConfigParserSuite) TestParseFromReader(c *gc.C) {
-	parsed, err := configparser.ParseReader(strings.NewReader("[DEFAULT]\ntesting = value\n\n[othersection]\nmyoption = myvalue\nnewoption = novalue\n\n[testing]\nmyoption = value\n\n"))
+
+	input := "[DEFAULT]\ntesting = value\n\n[othersection]\nmyoption = myvalue\nnewoption = novalue\nfinal = foo[bar]\n\n[testing]\nmyoption = value\n\n"
+	c.Logf("set input %s\n", input)
+	parsed, err := configparser.ParseReader(strings.NewReader(input))
 	c.Assert(err, gc.IsNil)
 
 	result, err := parsed.Items("othersection")
 	c.Assert(err, gc.IsNil)
-	c.Assert(result, gc.DeepEquals, configparser.Dict{"myoption": "myvalue", "newoption": "novalue"})
+	c.Assert(result, gc.DeepEquals, configparser.Dict{"myoption": "myvalue", "newoption": "novalue", "final": "foo[bar]"})
+
```

Comment:
Could probably tidy this last wee bit of whitespace up too?

Reply:
ah yes, thank you will tidy up and remove debug.  appreciate the quick response :)